src/commands/install.rs

```diff
@@ -39,19 +39,27 @@ pub fn install(options: InstallOpt) -> Result<(), failure::Error> {
     if !options.packages.is_empty() {
         let mut packages = vec![];
         for name in options.packages {
-            let q = GetPackageQuery::build_query(get_package_query::Variables {
-                name: name.to_string(),
-            });
-            let response: get_package_query::ResponseData = execute_query(&q)?;
-            let package = response
-                .package
-                .ok_or(InstallError::PackageNotFound { name: name.clone() })?;
-            let last_version = package
-                .last_version
-                .ok_or(InstallError::NoVersionsAvailable { name: name.clone() })?;
-            let package_name = package.name.clone();
-            let package_version = last_version.version.clone();
-            packages.push((package_name, package_version));
+            let name_with_version: Vec<&str> = name.split("@").collect();
```

Comment:
I hope I'm not stepping on any toes leaving some feedback here!

If I am please just let me know.

---

Should this `split` be limited to producing, at max, 2 items?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c44e48c5ec293c5c8aa8c227b99172be

What should be done in the case of `foo@bar@1.2.3` or `foo@1.2.3@bar`?

In the current implementation, the (invalid?) `@bar` suffix on the latter would be silently ignored instead of erroring out to the user something like "hey that isn't a valid spec, too many @-delimited components."